selfify/sidebar/sidebar.css

```diff
@@ -0,0 +1,6 @@
+
+.drop_zone {
```

Comment:
I think this should be # because its an id, not a class, in the HTML. The drop zone right now is really small.